### PR TITLE
LLama prompt

### PR DESCRIPTION
**Pull request by OskarStark**
Untested, but this could do the trick

### CODE REVIEW

File: src/Model/Language/Llama.php [hunk 1]
```diff
@@ -22,14 +26,37 @@ public function call(MessageBag $messages, array $options = []): TextResponse
...
     }
 
+    private static function convertToPrompt(MessageBag $messageBag): string
+    {
+        $messages = [];
+
+        /** @var MessageInterface $message */
+        foreach ($messageBag->getIterator() as $message) {
+            if ($message instanceof UserMessage) {
+                $content = $message->content[0]->text;
```
Comment (OskarStark) [inline src/Model/Language/Llama.php:45]: I think this is not enough here, because of the N-Contents we could have

**Comment by chr-hertel** [comment]
i'm afraid that's gonna be more complex:
https://www.llama.com/docs/model-cards-and-prompt-formats/llama3_1/#-supported-roles-

**Comment by OskarStark** [comment]
Ah I remember, I mixed sth. up, sorry.

**Comment by OskarStark** [comment]
Replaced by 
* #135

**Pull request closed without merging**